impl/src/main/java/org/ehcache/impl/internal/store/offheap/OffHeapMap.java

```diff
@@ -0,0 +1,23 @@
+package org.ehcache.impl.internal.store.offheap;
+
+/**
+ * Off-heap-backed maps counters.
+ *
+ * @author Ludovic Orban
+ */
+public interface OffHeapMap {
```

Comment:
Why the extra interface instead of sticking these on `EhcacheOffHeapBackingMap<K, V>`?